benchmarks/scripts/run_benchmarks.sh

```diff
@@ -54,14 +54,14 @@ QASM_EXPVAL_FILES=(
     "ucc/prep_select_N10_ghz_basis_rz_rx_ry_h_cx.qasm"
     "ucc/qcnn_N10_4layers_basis_rz_rx_ry_h_cx.qasm"
 )
-LOG_EXPVAL=true
+LOG_EXPVAL=false
 for qasm_file in "${QASM_EXPVAL_FILES[@]}"; do
     for compiler in "${COMPILERS[@]}"; do
         # Combine the common folder path with the QASM file
         full_qasm_file="${QASM_FOLDER}${qasm_file}"
         
         # Build the command, passing the results folder as an argument
-        command="python3 $(dirname "$0")/expval_benchmark.py \"$full_qasm_file\" \"$compiler\" \"$RESULTS_FOLDER\" $LOG_EXPVAL"
+        command="poetry run python $(dirname "$0")/expval_benchmark.py \"$full_qasm_file\" \"$compiler\" \"$RESULTS_FOLDER\" $LOG_EXPVAL"
```

Comment:
One thing to be careful about here is if this script itself is run via poetry, which is how I think the prior benchmark runs were done in the github actions (e.g. they were run via `poetry run ./benchmarks/scripts ....`. 

*update* Suprisingly, this seems to work!